src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int ZERO = 0;
+    static final int ONE = 1;
```

Comment:
don't name constants after their values, they may change - name them after the role they serve in the app. It's the opportunity to increase readability - tell to those who will read your code, what's going on
OPERATION_INDEX, AMOUNT_INDEX